app/src/main/res/layout-land/profile_progress_header.xml

```diff
@@ -74,13 +73,15 @@
       android:clickable="@{viewModel.ongoingTopicCount > 0}"
       android:importantForAccessibility="yes"
       android:onClick="@{() -> viewModel.clickOnOngoingTopicCount()}"
-      app:cardBackgroundColor="@color/component_color_profile_progress_activity_stories_progress_background_color"
+      app:cardBackgroundColor="@color/component_color_shared_rounded_rect_background_color"
       app:cardElevation="4dp"
       app:contentPadding="8dp"
       app:layout_constraintEnd_toStartOf="@+id/completed_stories_container"
       app:layout_constraintHorizontal_weight="1"
       app:layout_constraintStart_toEndOf="@+id/profile_edit_image"
-      app:layout_constraintTop_toBottomOf="@+id/profile_name_text_view">
+      app:layout_constraintTop_toBottomOf="@+id/profile_name_text_view"
+      app:strokeColor="@color/component_color_profile_progress_activity_story_count_card_stroke_color"
```

Comment:
As the `android:background="@drawable/grey_card_rounded_border"` this is not effecting any stroke/background. That's why I added this.

So we remove `android:background="@drawable/grey_card_rounded_border"` ?

Reply:
Yes you can remove it.

Reply:
Okay thanks.